demo/docker-compose.alice-bob.yml

```diff
@@ -33,3 +33,25 @@ services:
     ports:
       - "3003:3003"
     command: poetry run aca-py start --arg-file ./configs/bob.yml 
+
+  ngrok-tails-server:
+    image: wernight/ngrok
+#    networks:
+#      - tails-server
+    ports:
+      - 4044:4040
+    command: ngrok http tails-server:6543 --log stdout
```

Comment:
We should probably use localtunnel for this rather than ngrok. I don't think this is hugely significant for any reason other than consistency and the opportunity for you to rely on your own services when the public ones are unavailable but I think those two points are probably enough. The `agent-tunnel` image is using `localtunnel` behind the scenes and the `${AGENT_TUNNEL_HOST}` variable can be used to point to a self-hosted `localtunnel` server.

Reply:
Great point! I'll go ahead and make the change over to localtunnel instead. Great to know that localtunnel and ngrok are functionally equivalent in this regard.